static/js/compose.js

```diff
@@ -538,6 +544,13 @@ exports.transmit_message = function (request, success, error) {
         send_message_ajax(request, success, error);
     }
 };
+function report_content_not_sent(request) {
```

Comment:
It probably wouldn't be too hard to get test coverage that this function gets called if the message never transmits successfully.  You could export the function so that it's possible to stub it.